var/spack/repos/builtin/packages/py-proj/package.py

```diff
@@ -0,0 +1,27 @@
+from spack import *
```

Comment:
Is this a problem?  I find tons of packages with missing license info.

On Fri, Sep 2, 2016 at 9:25 AM, Adam J. Stewart <notifications@github.com>
wrote:

> In var/spack/repos/builtin/packages/py-proj/package.py:
>
> > @@ -0,0 +1,27 @@
> > +from spack import *
>
> Missing licensing info above.
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/LLNL/spack/pull/717/files/15797aace19e798d7a7cf80e7e3e345687672414#r77345182>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AB1cdw86zpWnQEg3R35UVDGdNMwDsChWks5qmCPfgaJpZM4H9_nz>
> .
>